Rubberduck.Resources/Inspections/InspectionResults.resx

```diff
@@ -513,4 +513,8 @@ In memoriam, 1972-2018</value>
     <value>The annotation '{0}' was expected to have less arguments.</value>
     <comment>{0} annotation name</comment>
   </data>
+  <data name="MisleadingByRefParameterInspection" xml:space="preserve">
+    <value>Misleading ByRef modifier used for Parameter '{0}'.</value>
+    <comment>{0} Parameter name</comment>
```

Comment:
Is the upper case for the word _parameter_ correct here?

Reply:
No...I'll modify it.